URBNSwiftAlert/Classes/AlertView.swift

```diff
@@ -126,6 +128,7 @@ extension AlertView {
         buttonsSV.distribution = .fillEqually
         buttonsSV.spacing = configuration.styler.button.spacing
         stackView.addArrangedSubviews(separatorBorderView, buttonsSV.wrapInNewView(with: configuration.styler.button.containerInsetConstraints))
+        separatorBorderView.bottomAnchor.constraint(equalTo: buttonsSV.topAnchor).isActive = true
```

Comment:
I wouldn’t just constrain this within the stack view. I would create a new stack view with 0 spacing and add these two views in them